spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -83,6 +83,23 @@ module XCPretty
 )
     end
 
+    it "formats uncaught exceptions" do
+      @formatter.format_uncaught_exception("testRaisingUncaughtException",
+      'NSInvalidArgumentException',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.